readme.md

```diff
@@ -129,44 +92,59 @@ and color), and others stay the same, for example, `fuel` starts at 100, and
 
 Why don't we build a function that makes these objects for us!
 
-### Exercise (10 minutes - 20/150)
+### You-Do Exercise: Create a makeCar function (10 min)
 
 Define a function: `makeCar()` that takes two parameters (model, color) and
 makes a new object literal for a car using those params, and returns that object.
 
-For example:
 ```js
 // This should return a car object just like the previous example
 var celica = makeCar("Toy-Yoda Celica", "limegreen");
 ```
-
 See solution in `car.js`
 
-## Constructor Functions (20 minutes - 30/150)
+
+## Constructor Functions
+
+### You-Do Exercise: Read Understanding JavaScript Constructors (5 min)
+
+[Understanding JavaScript Constructors](https://css-tricks.com/understanding-javascript-constructors/) 
+
+Read the First Few Paragraphs and STOP at **Object.defineProperty Function** Section! 
+
+<!--Please tilt laptops when finished reading -->
+
+#### Turn & Talk- 1/4: Why might we want to use a constructor function in OOP to create objects? (5 min)
+
+### Overview of Constructor Functions (10 min)
 
 It's so common that we need to make objects with similar properties / methods
-that programming languages usually have some features to help with this. In
-Ruby, we have **classes** which can create **objects**, or **instances** of
-that class.
+that programming languages usually have some features to help with this.
 
-Javascript doesn't really have classes, instead we use constructor functions and
-prototypes to achieve similar results. (Note: the newest version, ES6, adds
-classes, but they're really just syntactic sugar, behind the scenes it's still
-using constructors and prototypes).
+In Javascript, we use constructor functions and
+prototypes. ***(Note: the newest version, ES6, adds
+classes, but they're really just syntactic sugar, behind the scenes it's still using constructors and prototypes)***
 
 Constructor functions are a lot like the `makeCar` function we just created, but
 they're supported by JS and we use the `new` keyword to use the constructor
 function.
+<!--maybe demonstrate that constructor is a property, All objects inherit a constructor property from their prototype:-->
 
 Note that constructor functions start with a capital letter to make it obvious
 that they are constructors. This isn't necessary, but is a convention and you
 should follow it!
 
+## Break (10 min)
+
+### I-Do: Make a Person Constructor Function (10 min)
+
 ```js
 function Person(initialName) {
```

Comment:
what's the motivation of using the new keyword? why not just use functions that return object literals?